modules/govuk_jenkins/templates/jobs/data_sync_complete_integration.yaml.erb

```diff
@@ -22,6 +22,8 @@
            ssh deploy@calculators-frontend-1.frontend 'cd /var/apps/smartanswers ; govuk_setenv smartanswers bundle exec rake panopticon:register'
            # Queue up any publisher scheduled editions that have been transferred across.
            ssh deploy@backend-2.backend 'cd /var/apps/publisher ; govuk_setenv publisher bundle exec rake editions:requeue_scheduled_for_publishing'
+           # Publish any pre-production finders from Specialist Publisher Rebuild.
+           ssh deploy@backend-1.backend 'cd /var/apps/specialist-publisher-rebuild ; govuk_setenv specialist-publisher-rebuild bundle exec rake publishing_api:publish_finders rummager:publish_finders'
```

Comment:
updated as per Alex's suggestion (also tested this command on Jenkins manually).